script/install_chromium.sh

```diff
@@ -10,24 +10,35 @@ readonly TARGET_DIR=$1
 # The build of Chromium used to test web functionality.
 #
 # Chromium builds can be located here: https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/
-readonly CHROMIUM_BUILD=768968
-# The ChromeDriver version corresponding to the build above. See
-# https://chromedriver.chromium.org/downloads
-# for versions mappings when updating Chromium.
-readonly CHROME_DRIVER_VERSION=84.0.4147.30
+#
+# Check: https://github.com/flutter/engine/blob/master/lib/web_ui/dev/browser_lock.yaml
+readonly CHROMIUM_BUILD=929514
+
+# The correct ChromeDriver is distributed alongside the chromium build above, as
+# `chromedriver_linux64.zip`, so no need to hardcode any extra info about it.
```

Comment:
Do we know if this will be true in general going forward?

Reply:
This has been true since February 2019:

* https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/634372/ 

But I don't have any guarantees that this is going to continue to be true in the future (but it's very convenient as is!). Let me ask in YAQS, see if anybody responds.

Reply:
> This has been true since February 2019

That sounds like the kind of "temporary" that's not likely to change a lot, so it seems fine to depend on. It's not like it's hard to go back to the old approach if it's changed without something better (and it sounds like it would probably remain this way until there's something better).

Reply:
Yes, if this assumption changes, we'd also need to change the CIPD roller script in the engine repo (but that thing only runs seldomly, this script would break much earlier)